programmers/문자열-내-p와-y의-갯수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var answer = true;
+    var pcount = 0;
+    var ycount = 0;
```

Comment:
변수타입으로 var 사용은 지양하시는게 좋아요~ ES6이후 부터 도입된 let, const(상수) 타입을 이용해주세요!
```suggestion
    let answer = true;
    let pcount = 0;
    let ycount = 0;
```

비슷한 것끼리는 이렇게 비구조화 할당을 이용해서 선언하셔도 됩니다.
비구조화 할당: https://velog.io/@public_danuel/destructuring-assignment
```suggestion
    let answer = true;
    let [pcount, ycount] = [0, 0];
```